<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gvGraphics">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbBuild">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbScaleXPlus">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>540</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>X+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbScaleXMinus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>X-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbScaleYPlus">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Y+</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwFunc">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>211</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbScaleYPlus_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>460</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Y-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRight">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>410</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLeft">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>410</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDown">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>470</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Вниз</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUp">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>350</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Вверх</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblX1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblX2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblX3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblX4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblX5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>510</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblY5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>470</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblY4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblY3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblY2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblY1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>-10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwPx">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>30</y>
      <width>191</width>
      <height>192</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAdd">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить слогаемое</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRemove">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить последнего слогаемое</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P(X)=a0+x*a1+...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>F(P(X))</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
